lib/taskana-core/src/main/java/pro/taskana/task/api/models/Task.java

```diff
@@ -244,6 +244,13 @@ void setPrimaryObjRef(
    */
   String getClassificationCategory();
 
+  /**
+   * Returns the count of the comments of the current {@linkplain Task}.
```

Comment:
You don't need to link Task here because we are in the Task.java

Reply:
okay